portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,41 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Jennifer's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+    <header>
+      <h1>JENNIFER JIMENEZ</h1>
+    </header>
+    <nav>
+      <ul class="navbar">
+        <li>Home</li>
+        <li>Projects & Work</li>
+        <li>Gallery</li>
+      </ul>
+    </nav>
+    <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <p>
+        Hello! My name is Jennifer and I am rising junior at Yale University studying computer science.
+        I am a proud member of Trumbull College at Yale. On campus, I am involved with the Yale Undergraduate Prison 
+        Project as GED tutor for justice-impacted individuals.
+      <br></br>
+        I am originally from San Bernardino, California, which is where I am currently located. I love spending time
+        outdoors, whether just in my own yard, a nearby park, or some other nearby attraction. Being located in the Inland 
+        Empire means I have mountains, the beach, and the desert all within my reach, which I am very grateful for!
+      <br></br>
+        In my free time, I like to go on walks, do yoga, rewatch my favorite shows, and listen to music. I also
+        like to paint (I'm bad at it) and play guitar, though I don't do it as much as I would like to. During quarantine
+        I got into skateboarding a little bit and I would like to continue during the summer. :)
+      </p>
+      <img src="/images/jen_nyc.jpg" width = 300px />
+      <h2>Get to know me more!</h2>
+        <p>Click here to learn a random fact about me:</p>
+      <button onclick="addRandomGreeting()">Jen Fact!</button>
       <div id="greeting-container"></div>
+      <br></br>      
```

Comment:
nit: You should be able to remove this. If you need to increase margins you can do so in your CSS.